tqftpserv.c

```diff
@@ -47,6 +47,7 @@ struct tftp_client {
 	unsigned int timeoutms;
 	off_t seek;
 
+	uint8_t *blk_buf;
```

Comment:
Please start your commit message by describing the problem. What is currently broken and why.

Reply:
It is better to have a big commit message than the one not describing the issue.